tests/foreman/api/test_contentview.py

```diff
@@ -316,6 +317,36 @@ def test_negative_add_dupe_modules(self):
             ).create()
         self.assertEqual(len(content_view.read().puppet_module), 1)
 
+    @tier2
+    def test_positive_add_sha512_rpm(self):
+        """Associate sha512 RPM content in a view
+
+        :id: 1f473b02-5e2b-41ff-a706-c0635abc2476
+
+        :expectedresults: Custom sha512 assigned and present in content view
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Pulp
+
+        :CaseImportance: Medium
+
+        :BZ: 1639406
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        yum_sha512_repo = entities.Repository(product=product, url=CUSTOM_RPM_SHA_512).create()
+        yum_sha512_repo.sync()
+        content_view = entities.ContentView(organization=org).create()
+        self.assertEqual(len(content_view.repository), 0, content_view)
+        content_view.repository = [yum_sha512_repo]
+        content_view = content_view.update(['repository'])
+        content_view.publish()
+        self.assertEqual(len(content_view.repository), 1, content_view)
```

Comment:
Why is `content_view` message to display in case of failure? That's rather uncommon.

Reply:
I am used to add a `msg` to be displayed in case of test failure. Just to help troubleshooting. I can remove it.

Reply:
I see. In Robottelo, we usually do not set message and depend on pytest failure reporting.